libseccomp/src/error.rs

```diff
@@ -369,6 +369,37 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_with_msg() {
+        assert_eq!(
+            format!("{}", SeccompError::with_msg(TEST_ERR_MSG).msg()),
```

Comment:
Can we use `.as_str()` or `.to_string()` instead of `format!` here?

Reply:
Oh, I could just use `assert_eq!(SeccompError::with_msg(TEST_ERR_MSG).msg(), TEST_ERR_MSG);`.
Thanks!